optimizers/prioritized.js

```diff
@@ -0,0 +1,363 @@
+var path = require("path");
```

Comment:
OK no problem. Are you thinking optimization packages/functions should be hosted within the systemjs ecosystem or just on their creators' accounts?